pages/common/xml-select.md

```diff
@@ -0,0 +1,25 @@
+# xml select
+
+> Select from XML documents using XPATHs.
+> Tip: use `xml elements` to display the XPATHs of an XML document.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Select all elements matching "XPATH1" and print the value of the element "XPATH2":
+
+`xml select --template --match {{"XPATH1"}} --value-of {{"XPATH2"}} {{path/to/input.xml|URI}}`
+
+- Match  "XPATH1" and print the value of "XPATH2" as text with new-lines:
+
+`xml select --text --template --match {{"XPATH1"}} --value-of {{"XPATH2"}} --nl {{path/to/input.xml|URI}}`
+
+- Count the elements of "XPATH1":
+
+`xml select --template --value-of "count({{XPATH1}})" {{path/to/input.xml|URI}}`
+
+- Count all nodes in XML document(s):
```

Comment:
This way we would get rid of the `(s)`, but you can keep your version if you want to.
```suggestion
- Count all nodes in one or more XML documents:
```

Reply:
Will adopt